modules/color-picker/react/spec/ColorInput.spec.tsx

```diff
@@ -92,6 +92,15 @@ describe('ColorInput', () => {
     mount(<ColorInput inputRef={ref} value={''} />);
     expect(ref.current && ref.current.tagName && ref.current.tagName.toUpperCase()).toBe('INPUT');
   });
+
+  test('ColorInput should spread extra props', () => {
+    const component = mount(<ColorInput data-propspread="test" />);
+    const input = component
+      .find('input') // TODO: Standardize on prop spread location (see #150)
```

Comment:
Is this still a TODO? I didn't know how to grok this comment with the context

Reply:
Need to follow up on the non-standard behavior of all of these. Tracking in https://github.com/Workday/canvas-kit/issues/171